js/.eslintignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+dist
+lib
```

Comment:
It's a good practice to add a newline at the end of the file (.eslintignore) to avoid potential issues in different environments.